src/components/CampaignsAdmin/edit/CampaignDetails.tsx

```diff
@@ -87,6 +82,18 @@ export function CampaignDetails() {
     }
     return data;
   };
+  
+  /**
+     * Remove location from campaign
+     */
+  const removeAddedLocation = (locationId:number, locationCONF: CampaignLocationInfo[] | undefined) => {
```

Comment:
Why was this needed?

Reply:
ohhhh now I understand. I think its best to not physically remove rows from the database. We can mark them deleted instead and keep the record

Reply:
That functionality does not delete a location from the DB. It removes a location from the locations list assigned to a campaign. This is important because the user may add a location to a campaign by mistake.